drivers/flash/flash_stm32_ospi.c

```diff
@@ -935,8 +935,8 @@ static void flash_stm32_ospi_isr(const struct device *dev)
 	HAL_OSPI_IRQHandler(&dev_data->hospi);
 }
 
-/* weak function required for HAL compilation */
-__weak HAL_StatusTypeDef HAL_DMA_Abort_IT(DMA_HandleTypeDef *hdma)
+#if !defined(CONFIG_SOC_SERIES_STM32H7X)
+/* weak function required for HAL compilation */__weak HAL_StatusTypeDef HAL_DMA_Abort_IT(DMA_HandleTypeDef *hdma)
```

Comment:
```suggestion
#if !defined(CONFIG_SOC_SERIES_STM32H7X)
/* weak function required for HAL compilation */
__weak HAL_StatusTypeDef HAL_DMA_Abort_IT(DMA_HandleTypeDef *hdma)
```
Is slightly easier to read ;)

Reply:
yes, changed